recipes/finetuning_openai_summaries.ipynb

```diff
@@ -0,0 +1,1723 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    disk_cache = dc.Cache("cache")
</pre><p> </p><p>do we really need disk cache for this? Can't we use lru_cache directly?</p><p> </p><p>We should explain why we use a cache</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
it was only to accelerate the development, I am removing it